src/main/java/gregtech/common/blocks/GT_Block_Reinforced.java

```diff
@@ -180,6 +187,62 @@ public void dropBlockAsItemWithChance(World worldIn, BlockPos pos, IBlockState s
         }
     }
 
+    @Override
+    public boolean removedByPlayer(IBlockState state, World world, BlockPos pos, EntityPlayer player, boolean willHarvest) {
+        if(!world.isRemote && state.getValue(METADATA) == 5){
+            EntityTNTPrimed entitytntprimed = new EntityTNTPrimed(world, pos.getX() + 0.5F, pos.getY() + 0.5F, pos.getZ() + 0.5F, player);
+            world.spawnEntityInWorld(entitytntprimed);
+            world.playSound((EntityPlayer)null, pos, SoundEvents.ENTITY_TNT_PRIMED, SoundCategory.BLOCKS, 1.0F, 1.0F);
+
+            world.setBlockToAir(pos);
+            return false;
+        }
+        return super.removedByPlayer(state, world, pos, player, willHarvest);
+    }
+
+    @Override
+    public void onBlockAdded(World worldIn, BlockPos pos, IBlockState state) {
+        super.onBlockAdded(worldIn, pos, state);
+        if (worldIn.isBlockPowered(pos) && state.getValue(METADATA) == 5) {
+            this.removedByPlayer(state, worldIn, pos, null, false);
+        }
+    }
+
+    @Override
+    public void neighborChanged(IBlockState state, World worldIn, BlockPos pos, Block blockIn) {
+        if (worldIn.isBlockPowered(pos) && state.getValue(METADATA) == 5) {
+            this.removedByPlayer(state, worldIn, pos, null, false);
+        }
+    }
+
+    @Override
+    public void onBlockExploded(World world, BlockPos pos, Explosion explosion) {
+        if (!world.isRemote && world.getBlockState(pos).getValue(METADATA) == 5) {
+            EntityTNTPrimed entitytntprimed = new EntityTNTPrimed(world, (double)((float)pos.getX() + 0.5F), (double)pos.getY(), (double)((float)pos.getZ() + 0.5F), explosion.getExplosivePlacedBy());
+            entitytntprimed.setFuse(world.rand.nextInt(entitytntprimed.getFuse() / 4) + entitytntprimed.getFuse() / 8);
+            world.spawnEntityInWorld(entitytntprimed);
+        }
+        super.onBlockExploded(world, pos, explosion);
+    }
+
+    @Override
+    public boolean onBlockActivated(World worldIn, BlockPos pos, IBlockState state, EntityPlayer playerIn, EnumHand hand, @Nullable ItemStack heldItem, EnumFacing side, float hitX, float hitY, float hitZ) {
+        if (heldItem != null && (heldItem.getItem() == Items.FLINT_AND_STEEL || heldItem.getItem() == Items.FIRE_CHARGE) && state.getValue(METADATA) == 5) {
```

Comment:
I thought about it but there is did not see way to check if item has lighter behawior without adding method to GT_MetaBase_Item and 5.09 doesn't check either so I left it as is.